src/Form/DataTransformer/CitizenProjectCommitteeSupportTransformer.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace AppBundle\Form\DataTransformer;
+
+use AppBundle\Entity\CitizenProjectCommitteeSupport;
+use AppBundle\Repository\CommitteeRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Symfony\Component\Form\DataTransformerInterface;
+
+class CitizenProjectCommitteeSupportTransformer implements DataTransformerInterface
+{
+    private $committeeRepository;
+
+    public function __construct(CommitteeRepository $committeeRepository)
+    {
+        $this->committeeRepository = $committeeRepository;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function transform($collection)
+    {
+        $uuids = [];
```

Comment:
You can remove this initialization and `return $uuids ?? [];` instead.

Reply:
✔️